src/Emitter.ts

```diff
@@ -96,6 +103,14 @@ export class Emitter {
    */
   _playback: Map<Event['key'], any[][]> = new Map();
 
+  /**
+   * Apply debug and playback options that are passed into the constructor
+   */
+  constructor ({ debug, playback }: IEmitterConstructor = {}) {
```

Comment:
This is not how components are supposed to work.

The constructor is relegated to the whims of component lifecycles and not emitters. Because components extend the Emitter it is unfeasible to supply constructor params.

This has to be configured via instance properties, such as when extending a class eg.

```ts
class MyFancyThing extends Component<any> {
  playback = 100
}

```